lib/routes/lnurl/withdraw/lnurl_withdraw_handler.dart

```diff
@@ -16,6 +17,10 @@ Future<LNURLPageResult?> handleWithdrawRequest(
   BuildContext context,
   LnUrlWithdrawRequestData requestData,
 ) async {
+  if (requestData.maxWithdrawable.toInt() ~/ 1000 < liquidMinimumPaymentAmountSat) {
+    throw Exception("Payment is below Liquid network limits, $liquidMinimumPaymentAmountSat sats.");
```

Comment:
I wouldn't use teh term "liquid" in error messages to the user. Perhaps just "... bellow network limits..."

Reply:
Done 817b727cfcb2e82e6ce658b7a86fbc11769017e9